.github/workflows/tests.yml

```diff
@@ -41,6 +41,9 @@ jobs:
     - name: Creating Zulip API key
       run: echo -e "${{ secrets.ZULIP_API_KEY }}" >> download
 
+    - name: Creating SECRET_KEY 
+      run: echo -e "${{ SECRET_KEY }}" >> "test"
```

Comment:
There is no secret key defined.
Use Github Workflow secret_key in this.
and why are you appending it to the test?

Reply:
I'm appending "test" as a dummy key to make sure that the Github actions pass. I'll add Github workflow `secret.SECRET_KEY`.